.github/workflows/e2e-tests.yml

```diff
@@ -38,9 +38,17 @@ jobs:
           npx playwright install chromium --with-deps
 
       - name: Start up WordPress environment
+        env:
```

Comment:
This doesn't have to happen in this PR, but we should consider running against 3 versions:

- Lowest version
- Latest released version
- Trunk

This way we can stay ahead of any potential bugs on all the versions we support, and will support.

Reply:
Agreed, just a bummer because these tests are so slow